packages/tools/kadena-cli/src/services/config/config.service.ts

```diff
@@ -62,11 +71,11 @@ export interface IConfigService {
 export class ConfigService implements IConfigService {
   // eslint-disable-next-line @typescript-eslint/naming-convention
   private directory: string | null = null;
-  public constructor(
-    // eslint-disable-next-line @typescript-eslint/naming-convention, @typescript-eslint/parameter-properties
-    private services: Services,
-    directory?: string,
-  ) {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  private services: Services;
+
+  public constructor(services: Services, directory?: string) {
```

Comment:
just call i IServices. so u dont have to disable lint. ( yeah i don't like it either but, better hthen the lint error)

Reply:
the lint is about the variable name, it should be `_services` because its private.